hs-bindgen/examples/distilled_lib_1.h

```diff
@@ -0,0 +1,82 @@
+/**
+ * Distillation of some test cases from a proprietary C library
+ *
+ * TODO: This will need some adjustments before it can compile, it's just
+ * an illustration of required features for now.
+ */
+
+#ifdef __GNUC__
+#define PACK_START     _Pragma("pack(1)")
+#define PACK_FINISH    _Pragma("pack()")
+#elif
+#define PACK_START
+#define PACK_FINISH
+#endif
+
+#ifdef __GNUC__
+#define PACK_ENUM __attribute__((packed))
+#else
+#define PACK_ENUM
+#endif
+
+// Structs
+
+/* Some comment*/
+PACK_START
+typedef struct a_typedef_struct
+{
+    bool                      field_0;    /* field 0 comment */
+    uint8_t                   field_1;    /* field 1 comment */
+    uint16_t                  field_2;    /* field 2 comment */
+    uint32_t                  field_3;    /* field 3 comment */
+    another_typedef_struct_t  field_4;    /* struct field */
+    another_typedef_struct_t *field_5;    /* struct pointer */
+    void                     *field_6;    /* void pointer
```

Comment:
comment not closed?

Reply:
Ah, hadn't noticed that. Well-spotted, will fix.